sdk/python/feast/client.py

```diff
@@ -687,7 +690,10 @@ def get_online_features(
             Each EntityRow provided will yield one record, which contains
             data fields with data value and field status metadata (if included).
         """
-
+        warnings.warn(
```

Comment:
Does this warning run for every single call?

Reply:
yes for every call to get_online_features()

Reply:
That might overload the system. Can we try and run it only once?

Reply:
I think a way is to keep track of how many times we've warned and if the number of times is 1 then we stop warning. But I'm not sure if that's a good approach since users may sometimes not see the warning on the first time.

Reply:
We can have a flag on client ie `self._has_warned_entity_row_deprecation` so that we warn the first time they call `get_online_features` per script run, but not every time they call `get_online_features()`. Users will see the warning every time they run their script.